index.html

```diff
@@ -35,7 +35,7 @@
 
     <div class="jumbotron text-center" id="header">
         <div class="mobileLogo">
-            <img src="img/Symbol.png" id="keyboardSymbol" height="100" width="200" alt="CodeWith">
+            <img src="img/CodeWithSymbol.png" id="keyboardSymbol" height="100" width="200" alt="CodeWith">
             <h1>We are CodeWith</h1>
         </div>
 

```

Comment:
The alt text should probably represent the text in an image - maybe add the tagline to the alt text for the desktop logo